### ISSUE TITLE
language-service: empty template should not trigger error

### ISSUE THREAD

**Issue by kemsky** [angular/angular#16404]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
![empty_template](https://cloud.githubusercontent.com/assets/3826972/25524759/9d5c4286-2c14-11e7-8c49-8eec09a8524f.png)

**Expected behavior**
No error.

**What is the motivation / use case for changing the behavior?**
Component is perfectly valid even with empty template.

* **Angular version:** 4.1.0

* **Language:** [TypeScript 2.2.2]

**Comment by kemsky** [comment]
Workaround is to use `&nbsp;` as template.

**Comment by specialkk** [comment]
Any update on this?

Also, a better workaround is to put the empty html comment `<!---->` as the template

**Comment by trotyl** [comment]
Should have been closed by #20651?

**Comment by DerZyklop** [comment]
related: #23553
